sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -621,15 +614,13 @@ def test_table_not_empty_and_write_disposition_empty(
         schema=bigquery.TableSchema())
     client.tabledata.List.return_value = bigquery.TableDataList(totalRows=1)
     write_disposition = beam.io.BigQueryDisposition.WRITE_EMPTY
-    with self.assertRaises(RuntimeError) as exn:
+    with self.assertRaisesRegexp(
+        RuntimeError, r'Table project:dataset.table is not empty but write '
+                      r'disposition is WRITE_EMPTY.'):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
Let's add `^` and `$` and escape the '.' characters throughout.

Reply:
done